src/sessionManager.ts

```diff
@@ -234,4 +210,31 @@ export default function SessionManager(
             }
         }
     };
+
+    /**
+     * Checks if the session has expired based on the last event timestamp
+     * @param lastEventTimestamp - Unix timestamp in milliseconds of the last event
+     * @param sessionTimeout - Session timeout in minutes
+     * @returns true if the session has expired, false otherwise
+     */
+    function hasSessionTimedOut(lastEventTimestamp: number, sessionTimeout: number): boolean {
```

Comment:
This might be an edge case, since we `lastEventTimestamp` and `sessionTimeout` have default values, but I would add a guard clause here.